docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -0,0 +1,128 @@
+[id="verifying-the-image-service-adoption_{context}"]
+
+= Verifying the {image_service} adoption
+
+Verify that you successfully adopted your {image_service_first_ref} to the {rhos_long} {rhos_curr_ver} deployment.
+//kgilliga: Does this procedure apply to all backends, or do some verification steps only apply to specific backends?
```

Comment:
The example points to a Ceph backend, but in  general I think the steps are the same for all the backends (except when you check the existing data on the Ceph cluster).
We can change the title and point to the fact this is an example with a Ceph backend.
Is that the goal?

Reply:
@fmount The goal was to have a general verification procedure that applies to all backends. However, another option is to add a ".Verification" heading in each specific backend procedure and only include the steps that apply to that backend. If some of these steps apply to all backends, the steps will be repeated in each procedure, but that's okay.

Reply:
We can keep the Ceph bits, and I can rewrite the step to say something like: "If you use a Ceph backend, <do this>."